### DISCUSSION TITLE
is  Harbor vulnerable to log4j attack !??

### DISCUSSION THREAD

**Discussion by alimp5** [goharbor/harbor#16114] [General]
examples:
https://github.com/advisories/GHSA-jfh8-c2jp-5v3q
https://confluence.atlassian.com/kb/faq-for-cve-2021-44228-1103069406.html

**Comment by sokesekken** [comment]
agree, this would be nice to know, also, I see that trivy can find vournabilities when scanning images in harbor, but it would be nice to generate a list of all images with this vournability.

**Comment by niemand-de** [comment]
It's crucial to know if Harbor is affected by CVE-2021-44228.

Thanks for your comments in advance.

@ywk253100 @wy65701436 @reasonerjt @steven-zou @wknet123 @ninjadq

**Comment by mk3rnb8** [comment]
agree, it is an important issue and I'm keen to know if harbor is vulnerable.

**Comment by wy65701436** [comment]
Per our scanning result, this CVE does not impact Harbor OSS.

**Comment by YAMLcase** [comment]
Just the due dilligence I've performed, it seems not:
- log4j search in code only brings up this discussion and an issue.  
- 0% of Harbor code written in Java.
